assets/dev/js/editor/controls/wp_widget.js

```diff
@@ -50,7 +50,9 @@ ControlWPWidgetItemView = ControlBaseDataView.extend( {
 					}
 				}
 
-				elementor.hooks.doAction( 'panel/widgets/' + self.model.get( 'widget' ) + '/controls/wp_widget/loaded', self );
+				var widgetType = self.model.get( 'widget' );
```

Comment:
```suggestion
				const widgetType = self.model.get( 'widget' );
```

Reply:
@matipojo 
This was my initial thought, but the entire file uses `var` instead of `consts`.